test/quantization/test_quantized_op.py

```diff
@@ -2524,6 +2524,30 @@ def test_qconv3d_unpack(
                 (stride_d, stride_h, stride_w), (pad_d, pad_h, pad_w),
                 channelwise)
 
+
+class TestPadding(TestCase):
+    @given(batch_size=st.integers(1, 64),
+           channels=st.integers(1, 64),
+           width=st.integers(16, 128),
+           qtype=st.sampled_from(hu._ALL_QINT_TYPES))
+    def test_reflection_pad1d(self, batch_size, channels, width, qtype):
```

Comment:
Is this also tested for QNNPACK? Do you need to explicitly loop over qengines?

Reply:
Does it need qnnpack testing? It was not implemented in qnnpack, neither is it implemented in fbgemm. The implementation is purely in memory manipulation, and should be qengine agnostic